init_query.sql

```diff
@@ -43,8 +43,13 @@ LEFT JOIN
     utterances AS utt  -- utterance-level stats such as speech rate.
 ON
     phone.u_ID = utt.u_id
+LEFT JOIN
+	(
+	    SELECT stdev(u.log_speech_rate) AS speech_rate, AVG(u.log_speech_rate) AS avg_speech_rate FROM utterances as u
+    ) AS z
```

Comment:
@Tarotis We compute average and stdev over all utterances here.

Reply:
perfect, I can implement this for the other cases (word form frequency) as well I guess